pydis_site/apps/content/resources/guides/pydis-guides/contributing/sir-lancebot.md

```diff
@@ -41,16 +41,6 @@ The requirements for Docker are:
     * This is only a required step for linux. Docker comes bundled with docker-compose on Mac OS and Windows.
 
 ---
-
-# Fork the Project
-You will need your own remote (online) copy of the project repository, known as a *fork*.
-
-- [**Learn how to create a fork of the repository here.**](../forking-repository)
-
-You will do all your work in the fork rather than directly in the main repository.
-
----
-
 # Development Environment
 1. Once you have your fork, you will need to [**clone the repository to your computer**](../cloning-repository).
```

Comment:
We could hyperlink "fork" to `pydis-guides/contributing.md` here to provide a really quick and inline link somewhere users may have a question.